tests/file_yaml.rs

```diff
@@ -35,6 +37,35 @@ fn make() -> Config {
         .unwrap()
 }
 
+#[test]
+fn test_keys_with_periods_deserialize_serde_yaml() {
+    let map = "192.168.1.1: a string value";
+
+    let c: HashMap<String, String> = serde_yaml::from_str(map).unwrap();
+
+    assert_eq!(c.get("192.168.1.1").unwrap(), "a string value");
+}
+
+#[test]
+fn test_keys_with_periods_deserialize_yaml_rust() {
+    use yaml_rust::YamlLoader;
+
+    let map = "192.168.1.1: a string value";
+
+    let c = YamlLoader::load_from_str(map).unwrap().first().unwrap().clone();
+
+    assert_eq!(c["192.168.1.1"].as_str().unwrap(), "a string value");
+}
+
+#[test]
+fn test_keys_with_periods_deserialize() {
+    let c = make();
+
+    let s: Settings = c.try_deserialize().unwrap();
+
+    assert_eq!(s.ip_key, "a string value");
+}
+
 #[test]
 fn test_file() {
```

Comment:
This test fails because the files in `tests/Settings` are not updated for the new key ("192.168.1.1")!

Reply:
It is updated.  It fails due to #417